Code Review

ggotora opened the pull request.

ggotora committed:
setup linters config file

ggotora committed:
initial commit

ggotora committed:
initial commit index.html, assets, styles.css

IjayAbby committed:
adjust the footer link tag

IjayAbby committed:
increase from width

ggotora committed:
change pwd-key position

kitpao commented:
# 👋 Hi @ggotora  and @IjayAbby ,

:pencil: Friendy reminder: Remember it's important to have a professional repo so you can increase your job opportunities.
Please take a step back and read the lesson here https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/.
Sadly for now, your review has been marked as **invalid** because of the following reason:
- [ ] You are missing a short summary of your changes in the pull request.

Once you correct this, please submit a new code review and we will be glad to check it :)
___
##### 👉 If you have any doubts you can contact me in slack as @kitpao
##### 👉 You can find me in your completition form as Kitzia Paola Vidal Marroquin

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

IjayAbby commented:
In this development branch, we created a form using HTML & CSS which was the second Microverse Project in the HTML & CSS course.
The Project required us to create a replica of the SignUp form in Mint.com or alternatively Mailchimp.com

ggotora committed:
fixed css indent lint error

IjayAbby committed:
Update README.MD file.

IjayAbby commented:
Fixed the README.MD file and CSS file errors.

IjayAbby committed:
use beautify to format the code

krys2fa commented on index.html:
```diff
@@ -0,0 +1,61 @@
+                    Terms and have read and acknowledge our Privacy Statement.</p>
+            </div>
+            <p class="form-footer center">
+                Invisible reCAPTCHA by Google Privacy Policy and Terms of Use.
```
- [ ] [OPTIONAL] The words `Privacy Policy` and `Terms of Use` should be web links as seen on the [original site](https://accounts.intuit.com/signup.html)

krys2fa commented on index.html:
```diff
@@ -0,0 +1,61 @@
+                <input type="password" id="password" autocomplete="current-password">
+                <span class="pwd-key">&#128275;</span>
+                <button class="btn" type="submit"><span>&#128274;</span> Create Account</button>
+                <p class="consent zero-mgn">By clicking Create account, you agree to our
```
- [ ] [OPTIONAL] The words `Terms` and `Privacy Statement` should be web links as seen on the [original site](https://accounts.intuit.com/signup.html)

krys2fa reviewed (changes requested):
## Awesome work @ggotora and @IjayAbby 👏 
Your project is a really great clone of the original site with good use of semantic HTML as well as CSS classes. 💪 
Good job on implementing the changes suggested by the previous TSE as well. 👍 
Just a few more adjustments to complete the project.
### Changes Required ♻️ 

- [ ] Kindly fix all linter errors. For example, you are using an indentation of more than 2 spaces in your `style.css` file.

- [ ] Also, check out comments below to add links to a few words at the bottom of the form.

Please request for another review after implementing the changes.

Happy Coding 🖥️ 😃 
Chris

###### You can contact me on slack (@ Chris Amanor) for any clarification.

IjayAbby committed:
Fix errors
rename image to main-logo
add package.json file
configure Webhint
configure Stylelint
Fix HTML & CSS indentation errors

IjayAbby committed:
add id attribute to email input

IjayAbby committed:
Code adjustments
add web links from the original site
adjust password key lock

expjazz reviewed (approved):
## Status: Project Approved :green_circle: 

#### Great work Team :clap:

The design looks really great and you implemented the requested changes by my colleagues above. Go ahead and merge :100: 

Good Luck and Happy Coding :+1: :

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

IjayAbby commented:
The project has been approved and the configuration files are working perfectly.
The reviewers gave helpful information on the areas to be worked on.
